client/lib/types.ts

```diff
@@ -7,9 +7,14 @@ export interface TestResponse {
 }
 
 export interface QuestionResponse {
-    languages?: string[];
+    languages?: Languages[];
     title: string;
     points: number;
     description?: string;
     tests: TestResponse[];
 }
+
+export interface Languages {
+    language: string;
```

Comment:
After you complete [this change](https://github.com/basalt-rs/basalt-server/pull/20/files#r2034195829), this will need to be updated as well.